Model/CategoryType.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Model
+{
+    public enum CategoryType
+    {
+        LunchStarters,
```

Comment:
CategoryType is just the name of the category. It's a separate entity and we shouldn't mix it with MenuCard imo. So, it should be Lunch instead of LunchStarters.